interface/python_binding/LICENSE

```diff
@@ -0,0 +1,178 @@
+Apache License
```

Comment:
Why are you including the license here? Is this another GitHub repo that has been copied here?

Reply:
The LICENSE file is published to [Python Package Index](https://pypi.org).